src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/CrossTargetSubscriptionHostBase.cs

```diff
@@ -262,10 +262,12 @@ await subscriber.Value.InitializeSubscriberAsync(this, _activeConfiguredProjectS
 
                     foreach (var subscriber in Subscribers)
                     {
-                        subscriber.Value.AddSubscriptionsAsync(newProjectContext);
+                        Shell.ThreadHelper.JoinableTaskFactory.Run(() => subscriber.Value.AddSubscriptionsAsync(newProjectContext));
```

Comment:
ie We've regressed so many times around this code - i'd prefer not to change it unless it's fixing a bug that we're running into right now.